conda-forge.yml

```diff
@@ -1,3 +1,5 @@
+azure:
+   max_parallel: 1
```

Comment:
Docs on this are a bit sparse atm. Raised issue ( https://github.com/conda-forge/conda-forge.github.io/issues/1928 ) to track that